### PR TITLE
Fix "more than one parameter" parser bug

### PR DESCRIPTION
**Pull request by parttimenerd**

### CODE REVIEW

File: src/main/java/minijava/parser/Parser.java [hunk 1]
```diff
@@ -310,16 +311,10 @@ private void parseExpressionWithPrecedenceClimbing(int minPrecedence) {
 
   /** UnaryExpression -> PostfixExpression | (! | -) UnaryExpression */
```
Comment (sherter) [inline src/main/java/minijava/parser/Parser.java:312]: Could you add a test case that demonstrates the problem? Looking at the production, the existing code looks better than the new code ;) (Maybe the comment needs to change, too?)

Comment (parttimenerd) [reply src/main/java/minijava/parser/Parser.java:312]: You're correct. I thought that the (unecessary) recursion could become a problem in tests with lots of `!` in a row. But this isn't actually the case (newer test case shows this).

**Comment by sgraf812** [comment]
I already fixed this in the `parser-qcheck` branch. So, if we can discuss whether to merge that first, there will be less bugs we fix independently in the long rung.

**Comment by parttimenerd** [comment]
We can just use you're bug fix in the `parse-qcheck` branch and delete this PR.

**Pull request closed without merging**